src/main/java/com/WildernessPlayerAlarm/WildernessPlayerAlarmConfig.java

```diff
@@ -55,4 +55,17 @@ default Color flashColor()
 	{
 		return new Color(255, 255, 0, 70);
 	}
+
+	@ConfigItem(
```

Comment:
Would you change this to `position = 1`, move it up in the config, and increment the position of the settings that are then below?